assets/src/stories-editor/helpers/index.js

```diff
@@ -526,14 +526,14 @@ export const filterBlockAttributes = ( blockAttributes, blockType, innerHTML ) =
 /**
  * Wraps all movable blocks in a grid layer and assigns custom attributes as needed.
  *
- * @param {WPElement} element                  Block element.
- * @param {Object}    blockType                Block type object.
- * @param {Object}    attributes               Block attributes.
- * @param {number}    attributes.positionTop   Top offset in pixel.
- * @param {number}    attributes.positionLeft  Left offset in pixel.
- * @param {number}    attributes.rotationAngle Rotation angle in degrees.
- * @param {number}    attributes.width         Block width in pixels.
- * @param {number}    attributes.height        Block height in pixels.
+ * @param {Element} element                  Block element.
```

Comment:
So this isn't a React element (with that weird `{import('react').ReactElement}` syntax), but a... DOM element?

Reply:
Indeed looks like it should be annotated as a React element:

https://github.com/WordPress/gutenberg/blob/403fa51163f2122017625ca94f852fb4c863d145/packages/blocks/src/api/serializer.js#L104-L111

As for the syntax, I was worried that a normal import would result in unused variable warnings. But it looks like that isn't the case, so I'll update that.